smrf/distribute/wind.py

```diff
@@ -590,6 +591,19 @@ def convert_wind_ninja(self, t):
         g_vel = utils.grid_interpolate(data_vel_int, self.vtx,
                                        self.wts, self.X.shape)
 
+        if np.any(np.isnan(g_vel)):
+            # https://stackoverflow.com/questions/9537543/replace-nans-in-numpy-array-with-closest-non-nan-value
+            self._logger.warning('NaN values present in the wind ninja interpolation, replaced with closest value')
+            for i in range(4):
+                mask = np.isnan(g_vel)
+                g_vel[mask] = np.interp(np.flatnonzero(mask), np.flatnonzero(~mask), g_vel[~mask])
+
+                if not np.any(np.isnan(g_vel)):
+                    break
```

Comment:
If this is intended for the outside of arrays, perhaps at this point it would be better at this point to have explicit constraints for that case?